Kudu.Core/Helpers/PostDeploymentHelper.cs

```diff
@@ -348,6 +349,47 @@ public static async Task PerformAutoSwap(string requestId, TraceListener tracer)
             }
         }
 
+        /// <summary>
+        /// Remove all site workers after cloudbuilt content is uploaded
+        /// </summary>
+        /// <param name="websiteHostname">WEBSITE_HOSTNAME</param>
+        /// <param name="sitename">WEBSITE_SITE_NAME</param>
+        /// <exception cref="ArgumentException">Thrown when RemoveAllWorkers url is malformed.</exception>
+        /// <exception cref="HttpRequestException">Thrown when request to RemoveAllWorkers is not OK.</exception>
+        public static async Task RemoveAllWorkersAsync(string websiteHostname, string sitename)
+        {
+            // Generate URL encoded auth token
+            string websiteAuthEncryptionKey = System.Environment.GetEnvironmentVariable(SettingsKeys.AuthEncryptionKey);
+            DateTime expiry = DateTime.UtcNow.AddMinutes(5);
+            string authToken = SimpleWebTokenHelper.CreateToken(expiry, websiteAuthEncryptionKey.ToKeyBytes());
+            string authTokenEncoded = HttpUtility.UrlEncode(authToken);
+            Trace(TraceEventType.Information, "RemoveAllWorkers, websiteAuthEncryptionKey = {0}", websiteAuthEncryptionKey);
+            Trace(TraceEventType.Information, "RemoveAllWorkers, authToken = {0}", authToken);
+            Trace(TraceEventType.Information, "RemoveAllWorkers, authTokenEncoded = {0}", authTokenEncoded);
+
+            // Generate RemoveAllWorker request URI
+            /* TODO: Hazhzeng, uncomment this when "RemoveAllStandardWorker" is out in ANT 83
+             * string baseUrl = $"http://{websiteHostname}/operations/removeworker/{sitename}/allStandard?token={authTokenEncoded}";
+             */
+            string baseUrl = $"http://{websiteHostname}/operations/removeworker/{sitename}/all?token={authTokenEncoded}";
```

Comment:
Maybe https?

Reply:
The internal call only accepts http protocol. Tried with https, failed to create SSL channel :(

Reply:
Fair